### ISSUE TITLE
{Freeway-Bug} [Adaptive Card > Application Login]: Focus order is not in sequential order while navigating in Application login page. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7598]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.20220628.3)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 15.5
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "ApplicationLogin.json" control and double tap to activate it.
7. Navigate the 'ApplicationLogin' page and observe the issue.

**Actual Result:**
Focus order is not in sequential order while navigating the Application login page as on performing right swipe on 'UserName' it is going on the 'Password' after that it is going on the edit fields.

**Expected Result:**
Focus order should be in sequential order and on performing right swipe on the 'UserName' label it should be land on the 'Username edit field' after that on performing right swipe again it should be go on the 'Password label' after that it should go on 
Password edit field.

**User Experience:**
It will impact the screen reader user as it will take extra effort for the user to navigate to all controls on the screen.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.04.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176612162-e5b106f2-505b-4f51-82ca-8f8a3e3e4d3b.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is not repro on the environment below, hence closing this issue.

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over
